applications/plugins/picopass/picopass_worker.c

```diff
@@ -406,6 +438,84 @@ ReturnCode picopass_write_card(PicopassBlock* AA1) {
     return ERR_NONE;
 }
 
+ReturnCode picopass_write_block(PicopassPacs* pacs, uint8_t blockNo, uint8_t* newBlock) {
```

Comment:
a lot of duplication with `write`, but not something I want to clean up right now